app/src/main/res/values-ja/strings.xml

```diff
@@ -0,0 +1,216 @@
+    <string name="archive_thumb" translatable="false">thumb</string>
+    <string name="archive_label" translatable="false">name</string>
+
+    <string name="random">ランダム</string>
+    <string name="title_activity_settings">設定</string>
+
+    <!-- Example General settings -->
+    <string name="pref_header_general">全般</string>
+
+
+    <string name="server_address_preference" translatable="false">server_address</string>
+    <string name="server_location_name">サーバーアドレス</string>
+    <string name="clear_cache">キャッシュをクリアしています…</string>
+
+    <string name="load_previews">より多くの親指をロード</string>
+    <string name="bookmark">ブックマーク</string>
+    <string name="unbookmark">ブックマークを外す</string>
+    <string name="archive_details">詳細</string>
+    <string name="archive_gallery">親指</string>
+    <string name="read_button">読む</string>
+    <string name="tag_header">タグ</string>
+    <string name="bookmark_label">ブックマーク</string>
+    <string name="detail_menu_item">詳細</string>
+    <string name="api_key_pref" translatable="false">api_key</string>
+    <string name="api_key_pref_name">APIキー</string>
+    <string name="details_title">詳細</string>
+    <string name="thumbs_title">親指</string>
+    <string name="license_key" translatable="false">ライセンス</string>
+    <string name="gpl_key" translatable="false">gpl</string>
+    <string name="git_key" translatable="false">github</string>
+    <string name="thumbnail_setting">サムネイルキャッシュをクリアする</string>
+    <string name="thumbnail_pref" translatable="false">clear_thumbs</string>
+    <string name="thumbnail_summary">タップしてサムネイルキャッシュを削除</string>
+    <string name="press_for_tags">タグを長押し</string>
+    <string name="first_time_text">タップしてサーバー設定をセットアップします。</string>
+    <string name="new_checkbox">新しい</string>
+    <plurals name="archive_count">
+        <item quantity="one">%dアーカイブ</item>
+        <item quantity="other">%dアーカイブ</item>
+    </plurals>
+    <string name="failed_image">読み込みに失敗しました</string>
+    <string name="verbose_pref" translatable="false">verbose_key</string>
+    <string name="verbose_pref_name">詳細なメッセージ</string>
+    <string name="verbose_pref_summary">より詳細なエラーメッセージを有効にする</string>
+    <string name="sort_method">ソート方法</string>
+    <string name="alpha_sort">名前</string>
+    <string name="sort_date">追加日</string>
+    <string name="save_button">保存</string>
+    <string name="sort_menu">ソート</string>
+    <string name="sort_pref" translatable="false">sort_pref</string>
+    <string name="sort_asc">昇順</string>
+    <string name="sort_desc">降順</string>
+    <string name="sort_direction">方向</string>
+    <string name="desc_pref" translatable="false">desc_pref</string>
+    <string name="tag_menu">タグ</string>
+    <string name="goto_page">ページに移動</string>
+    <string name="refresh_menu_item">強制更新</string>
+    <string name="rtol_pref_key" translatable="false">right to left pref</string>
+    <string name="rtol_pref_name" translatable="false">Right to Left Reader</string>
```

Comment:
This one was my mistake.  It is translatable.  I've updated the original strings.xml to show this.